core/src/main/scala/io/projectglow/sql/expressions/LogisticRegressionGwas.scala

```diff
@@ -43,6 +43,10 @@ case class LogitTestResults(
 
 object LogitTestResults {
   val nanRow: InternalRow = InternalRow(NaN, NaN, ArrayData.toArrayData(Seq(NaN, NaN)), NaN)
+  lazy val schema: StructType = ScalaReflection
```

Comment:
Is this only used in test code? If so please move to the test package.

Reply:
Btw, I don't think it's really any better to use `ScalaReflection` than `ExpressionEncoder`. It's an equally private interface.

Reply:
This seems to be publicly documented: https://kb.databricks.com/data/schema-from-case-class.html